test/linode_client_test.py

```diff
@@ -91,3 +91,22 @@ def test_create_client_with_label(self):
 
             self.assertEqual(m.call_url, '/longview/clients')
             self.assertEqual(m.call_data, {"label": "test_client_1"})
+
+    def test_get_subscriptions(self):
+        """
+        Tests that Longview subscriptions can be retrieved
+        """
+        r = self.client.longview.get_subscriptions()
+
+        self.assertEqual(len(r), 4)
+
+        expected_results = (
+            ("longview-10", "Longview Pro 10 pack"),
+            ("longview-100", "Longview Pro 100 pack"),
+            ("longview-3", "Longview Pro 3 pack"),
+            ("longview-40", "Longview Pro 40 pack"),
+        )
+
+        for result, (expected_id, expected_label) in zip(r, expected_results):
```

Comment:
this was a little weird to me, if only because I've rarely seen zip used before... any argument for the below snippet instead?

```

```
for result in expected_results:
    key = "".format(result)
    self.assertEqual(r[key], expected_results[key])
```

Reply:
mm that definitely doesn't work... h/o